__mocks__/schemas/ComprehensiveSourceSchema.json

```diff
@@ -368,17 +391,54 @@
                 "name": "Simple",
                 "fullName": "Simple",
                 "parentKey": "/ns0:SourceSchemaRoot/Looping/ManyToMany",
-                "schemaNodeDataType": "None",
                 "normalizedDataType": "ComplexType",
-                "properties": "NotSpecified",
-                "children": []
+                "properties": "Repeating",
```

Comment:
nit: can we call the target nodes something different than the source nodes? It makes debugging a little easier